src/wallet/walletActions.js

```diff
@@ -0,0 +1,20 @@
+import Promise from 'bluebird';
+
+export const GET_WALLET = '@wallet/GET_WALLET';
+export const GET_WALLET_START = '@wallet/GET_WALLET_START';
+export const GET_WALLET_SUCCESS = '@wallet/GET_WALLET_SUCCESS';
+export const GET_WALLET_ERROR = '@wallet/GET_WALLET_ERROR';
+
+export const getWallet = (username) => {
```

Comment:
@nil1511 the problem is the first scope is lost in your example. We can not easily see if the first one is the parent function since we can not nest the second one. I think this rule is much better for single line results or jsx (which naturally has clear scope) but not for multiple lines of JS.

I don't have a strong opinion on this but I just think it's not wrong in both cases and since I already wrote a lot of actions in this style I prefer to keep it this way (unless you believe it's wrong?)